### PR TITLE
making the routes more resful <tom>

### PR DESCRIPTION
**Pull request by biotom**
I've changed the route names around to make them more RESTful- I dont have an updated swagger spec yet, but I've raised a ticket with swagger inspector, so hopefully they'll be able to fix whatever issue I'm seeing.

### CODE REVIEW

File: knowledge_graph/routes.py [hunk 1]
```diff
@@ -6,29 +6,31 @@
...
 
 
 @app.route('/', methods=['GET'])
-def home():
-    return "<h1>API for climatemind ontology</h1>"
+def home() -> Tuple[str, int]:
+    return "<h1>API for climatemind ontology</h1>", 200
 
 
 @app.route('/ontology', methods=['GET'])
```
Comment (znurgl) [inline knowledge_graph/routes.py:32]: it is usually plural, especially if the response is a list. /ontologies

Comment (biotom) [reply knowledge_graph/routes.py:32]: Sure, but there's only one ontology- I think 'ontologies' would be misleading.

Comment (biotom) [reply knowledge_graph/routes.py:32]: My thinking here was that you're querying (via the query string) the ontology

**Comment by seanmajorpayne** [comment]
I think singular for ontology is good.

**Review by znurgl — APPROVED** [review]

**Pull request merged**